### PR TITLE
Fix #4609 : How to enable RTL

### LINKED ISSUE #4609: [Developer Video] How to enable RTL and check for issues in the app?
We should create a youtube video for `How to enable RTL and check for issues in the app?` so that contributors can easily follow the video for better understanding.

### PR DESCRIPTION
**Pull request by MohitGupta121**
## Explanation

Fix #4609 : How to enable RTL

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

### CODE REVIEW

**Review by MohitGupta121 — COMMENTED** [review]
@adhiamboperes PTAL, Thanks!

File: wiki/RTL-Guidelines.md [hunk 1]
```diff
@@ -19,6 +20,15 @@ In general, the passage of time is depicted as left to right for LTR languages,
 
 When a UI is changed from LTR to RTL (or vice-versa), it’s often called mirroring. An RTL layout is the mirror image of an LTR layout, and it affects layout, text, and graphics.
 
+# How to enable RTL?
```
Comment (adhiamboperes) [inline wiki/RTL-Guidelines.md:23]: There is a second option of setting the phone language to an RTL language like Arabic. What  do you think about adding that?

**Review by adhiamboperes — CHANGES_REQUESTED** [review]
Thanks @MohitGupta121! I have a suggestion but let me know what you think

File: wiki/RTL-Guidelines.md (hunk 1 above)
Comment (MohitGupta121) [reply wiki/RTL-Guidelines.md:23]: DONE.

**Review by MohitGupta121 — COMMENTED** [review]
@adhiamboperes PTAL, Thanks!

**Review by adhiamboperes — APPROVED** [review]
LGTM, thanks @MohitGupta121!

**Pull request merged**